### PR TITLE
Create script to review bot PRs

### PR DESCRIPTION
**Pull request by tim-s-ccs**
- Identical to the script for dependabot but changed to allow for multiple bot PRs to be reviewed

As we get PRs for snyk as well as dependabot, I thought it might be worth refactoring the script so that it could review PRs for multiple bots.

The only differences between dependabot PRs and snyk PRs are that they have different authors and slightly different labelling. The config for these bots is kept in the `BOTS_CONFIG` and this is used to differentiate them.

Other than that, the script remains unchanged other than exiting if one of the options given is not in the `BOTS_CONFIG`.

### CODE REVIEW

File: scripts/review-bot-prs.py [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
+    - dependabot
+    - snyk
 """
 
+import sys
 import json
 import subprocess
+from docopt import docopt
+
+sys.path.insert(0, '.')
```
Comment (bjgill) [inline scripts/review-bot-prs.py:20]: Is this needed? I think we only need to add this when we're importing from somewhere else within this repo.

File: scripts/review-bot-prs.py [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
 ORGANISATION = "Crown-Commercial-Service"
 
+BOTS_CONFIG = {
+    'dependabot': {
+        'login': 'dependabot',
+        'label_str': 'label:dependencies'
+    },
+    'snyk': {
+        'login': 'snyk-bot',
+        'label_str': 'author:snyk-bot'
```
Comment (bjgill) [inline scripts/review-bot-prs.py:31]: Cunning :tada:

**Review by bjgill — COMMENTED** [review]
A great idea. Just a few questions about the implementation details.

File: scripts/review-bot-prs.py [hunk 3]
```diff
@@ -73,6 +95,20 @@ def eligible_for_semiautomated_merge(pr):
...
+
+    BOT = arguments['<bot>']
+
+    if BOT not in BOTS_CONFIG:
+        print("FAILED: bot not in list, available bots are:")
+
+        for bot in BOTS_CONFIG:
+            print(f"  - {bot}")
+
+        exit(1)
```
[the remark this answers is not in the document]
Comment (tim-s-ccs) [reply scripts/review-bot-prs.py:108]: I did not know we could do this, I will implement it

**Comment by tim-s-ccs** [comment]
> A great idea. Just a few questions about the implementation details.

Thank you for the feedback, I have implemented the changes suggested

**Pull request merged**